README.md

```diff
@@ -524,3 +545,6 @@ optional boolean, suppresses the default console logging on successful authentic
 
 ###progress
 optional boolean, suppresses writing progress bars to stdout if set to `false`
+
+###requestOptions
+adds ability to pass default options for [request](https://github.com/request/request) module
```

Comment:
good to link to https://github.com/request/request#requestdefaultsoptions here, too.